kolibri/core/assets/src/api-resources/task.js

```diff
@@ -27,7 +27,7 @@ class TaskResource extends Resource {
 
 // TODO: switch to Model.delete()
   clearTask(taskId) {
-    const clientObj = { path: this.clearTaskUrl(), entity: { id: taskId } };
+    const clientObj = { path: this.clearTaskUrl(), entity: { task_id: taskId } };
```

Comment:
I thought we were thinking to just be more RESTful and do this as a `DELETE` on `/task/555/`? Ah, I guess that's the TODO in the comment above this? (Note: @benjaoming has banned TODO's in the code :D -- opening issues will ensure it gets tracked and triaged -- or, in this case, should be a fairly simple change)